### DISCUSSION TITLE
Transactions support in mongoose

### DISCUSSION THREAD

**Discussion by kathirsvn** [stargate/stargate-mongoose#21] [General]
Do we need to support transactions in stargate-mongoose like transactions in mongoose (https://mongoosejs.com/docs/transactions.html)? The last time, when we discussed about this @amorton mentioned below.

>  _We do not support Mongo style transactions, the better use of CQL LWT is in scope though_

@amorton Could you please elaborate a little bit more on this?

**Comment by amorton** [comment]
@kathirsvn we do not support those transactions, we do not have any support for multi documents transactions. We should throw whatever our unsupported error is.

**Reply by vkarpov15** [reply]
I agree, I don't think transaction support is important enough to prioritize for an alpha or even a first production release.

**Reply by kathirsvn** [reply]
Thanks @amorton and @vkarpov15 !
